src/ads/vast/client/VASTClient.js

```diff
@@ -208,25 +156,65 @@ VASTClient.prototype._buildAd = function buildAd(adJxonTree) {
     var inLine = ad.inLine;
 
     if (inLine && wrapper) {
-      throw new VASTError('on VASTClient._buildAd, InLine and Wrapper both found on the same Ad', 101);
+      return new VASTError('on VASTClient.getVASTAd.validateAd, InLine and Wrapper both found on the same Ad', 101);
     }
 
     if (!inLine && !wrapper) {
-      throw new VASTError('on VASTClient._buildAd, nor wrapper nor inline elements found on the Ad', 101);
+      return new VASTError('on VASTClient.getVASTAd.validateAd, nor wrapper nor inline elements found on the Ad', 101);
     }
 
     if (inLine) {
       if (inLine.creatives.length === 0) {
-        throw new VASTError("on VASTClient._buildAd, missing creative in InLine element", 101);
+        return new VASTError("on VASTClient.getVASTAd.validateAd, missing creative in InLine element", 101);
       }
     }
 
     if (wrapper) {
       if (!wrapper.VASTAdTagURI) {
-        throw new VASTError("on VASTClient._buildAd, missing 'VASTAdTagURI' in wrapper", 101);
+        return new VASTError("on VASTClient.getVASTAd.validateAd, missing 'VASTAdTagURI' in wrapper", 101);
```

Comment:
The Idea is to be able to know where the error happened by just looking at the error message.
How would your rewrite it?